console/views.py

```diff
@@ -14,56 +17,63 @@
 from console.database import db, SurveyResponse
 from console import app
 from console import settings
-from console.encrypter import Encrypter
-from console.helpers.exceptions import ClientError, ServiceError
-from console.queue_publisher import QueuePublisher
+from console.helpers.exceptions import ClientError, ResponseError, ServiceError
 
 
 logger = wrap_logger(logging.getLogger(__name__))
 
 
+@app.errorhandler(ResponseError)
+def handle_invalid_usage(error):
+    json_error = {"message": error.message, "status_code": error.status_code}
+    return jsonify(json_error)
+
+
 @app.route('/', methods=['GET'])
-@flask_security.login_required
 def home():
-    return "stuff"
+    return "home"
 
 
-def send_data(url, data=None, request_type="POST"):
+def send_data(logger, url, data=None, json=None, request_type="POST"):
+    response_logger = logger.bind(url=url)
     try:
         if request_type == "POST":
-            logger.info("Posting data to " + url)
-            r = requests.post(url, data)
-        else:
-            logger.info("Sending GET request to " + url)
+            response_logger.info("Sending POST request")
```

Comment:
Could/should this be something like 'Sending POST request to + url'?  Genuine question because it used to give the url in the log so you would've  had a reason to remove it.